courses/machine_learning/intro_to_tensorflow/3_keras_sequential_api.ipynb

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
<p>line 9: "create" -&gt; "Create"</p><p>line 13: I'd remove the spaces between the beginning of the line and "}"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Ok. Sounds goood.